storage/postgres/client.go

```diff
@@ -237,21 +237,7 @@ func (c *Client) listIndexerTables(ctx context.Context) ([]string, error) {
 	return tables, nil
 }
 
-// Wipe removes all contents of the database.
-func (c *Client) Wipe(ctx context.Context) error {
-	tables, err := c.listIndexerTables(ctx)
-	if err != nil {
-		return err
-	}
-	for _, table := range tables {
-		c.logger.Info("dropping table", "table", table)
-		if _, err = c.pool.Exec(ctx, fmt.Sprintf("DROP TABLE %s CASCADE;", table)); err != nil {
-			return err
-		}
-	}
-
-	// List, then drop all custom types.
-	// Query from https://stackoverflow.com/questions/3660787/how-to-list-custom-types-using-postgres-information-schema
+func (c *Client) listIndexerTypes(ctx context.Context) ([]string, error) {
```

Comment:
That's `listNexusTypes` to you, mister :roll_eyes: 
https://github.com/oasisprotocol/nexus/pull/467

Reply:
renamed! 🎨